### PR TITLE
Command to disable CI Builds

### PR DESCRIPTION
**Pull request by HelenaEksler**
#321

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@amitaibu I can check with dump, but I'll have not to sanitize github tokens (otherwise I won't get response from github whether .shoov.yml file exists).

**Comment by amitaibu** [comment]
sure, np.

**Comment by HelenaEksler** [comment]
[edit: removed image]

**Comment by amitaibu** [comment]
Can you also add the bash file to execute it

**Comment by HelenaEksler** [comment]
I see, the previous bash script was 305.sh. This should be 306.sh? Or other number? Or name doesn't matter?

**Comment by amitaibu** [comment]
issue number.

**Comment by HelenaEksler** [comment]
@amitaibu I've added bash script in the project root directory, probably you would like to move it to some other place. Drush script is in the `/upgrade_scripts` directory.

**Comment by HelenaEksler** [comment]
@amitaibu Can you advice how to deal with memory limit. For now we have not a lot CI Builds so we won't have this problem. But when we will have more - probably memory problem can appear. Since we have https://github.com/shoov/shoov-nodejs/issues/51 merged - command can go only over enabled builds with error "Incident Status". And when disable CI Build - set "Incident Status" to  none.

**Comment by amitaibu** [comment]
- `--range=50`
- `variable_set()` - remember the last processed item

**Comment by HelenaEksler** [comment]
@amitaibu Ready for review.

File: shoov/modules/custom/shoov_ci_build/drush/shoov_ci_build.drush.inc [hunk 1]
```diff
@@ -139,6 +145,56 @@ function drush_shoov_ci_build_check_nodejs_server_health() {
...
+  }
+  $result = $query
+    ->range(0, $range)
+    ->execute();
+  if (empty($result['node'])) {
+    drush_print('No CI Builds items left to check');
+    variable_set('shoov_ci_build_last_checked_for_config_file', 0);
+    return;
+  }
+  $ids = array_keys($result['node']);
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/drush/shoov_ci_build.drush.inc]: better node_load_multiple()

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**